src/main/java/edu/cornell/kfs/coa/batch/service/impl/CreateClosedLaborAccountsCsvServiceImpl.java

```diff
@@ -0,0 +1,241 @@
+            }
+            
+        } else {
+            LOG.error("getClosedLaborAccountsDateRange: Unknown Type of Closed Account File to Create = " + typeOfClosedAccountFileToCreate.get(CuCoaBatchConstants.ClosedLaborAccountsFileCreationConstants.FILE_DATA_CONTENT_TYPE_IS));
+        }
+        return fromToRange;
+    }
+    
+    private Date getJavaSqlDateForComputedDate(int daysPrevious) {
+        LocalDate computedFromDate = computeDateBeingDaysPreviousFromToday(daysPrevious);
+        java.sql.Date sqlDate;
+        try {
+            sqlDate = getDateTimeService().convertToSqlDate(computedFromDate.toString());
+        } catch (ParseException pe) {
+            LOG.error("getJavaSqlDateForComputedDate: ParseException caught for computedFromDate = " + computedFromDate.toString());
+            sqlDate = null;
+        }
+        return sqlDate;
+    }
+    
+    private LocalDate computeDateBeingDaysPreviousFromToday(int daysPrevious) {
+        LOG.info("computeDateBeingDaysPreviousFromToday: daysPrevious = " + daysPrevious);
+        DateTime todayAsDateTime = new DateTime(getDateTimeService().getCurrentDate());
+        DateTime datePreviousAsDateTime = todayAsDateTime.minusDays(daysPrevious);
+        LOG.info("computeDateBeingDaysPreviousFromToday: datePreviousAsDateTime = " + datePreviousAsDateTime.toLocalDate().toString());
+        return datePreviousAsDateTime.toLocalDate();
+    }
+    
+    private void writeClosedLaborAccountsToCsvFormattedFile(Map<String, Date> fileDataRangeSpecified) throws IOException {
+        LOG.info("writeClosedLaborAccountsToCsvFormattedFile: Prior to obtaining data with SQL query for date range. This may take a while.");
+        List<LaborClosedAccount> laborClosedAccountsData = getClosedLaborAccountsByDateRangeDao().obtainLaborClosedAccountsDataFor(fileDataRangeSpecified);
+        
+        LOG.info("writeClosedLaborAccountsToCsvFormattedFile: Have the data. Will now write it out.");
+        writeClosedAccountsToCsvFile(laborClosedAccountsData);
+    }
+    
+    private void writeClosedAccountsToCsvFile(List<LaborClosedAccount> laborClosedAccountsDataList) throws IOException {
+        String fullyQualifiedOutputFileName = generateFullyQualifiedOutputFileName();
+        LOG.info("writeClosedAccountsToCsvFile: fullyQualifiedOutputFile = " + fullyQualifiedOutputFileName);
+        File outputFile = new File(fullyQualifiedOutputFileName);
+        FileWriter outputFileWriter;
+        try {
+            outputFileWriter = new FileWriter(outputFile);
+            for (LaborClosedAccount closedAccount : laborClosedAccountsDataList) {
+                outputFileWriter.write(closedAccount.toCsvString());
+                outputFileWriter.write(KFSConstants.NEWLINE);
+            }
+            outputFileWriter.close();
+        } catch (IOException io) {
+            LOG.error("writeClosedAccountsToCsvFile: Caught IOException = " + io.getStackTrace().toString());
```

Comment:
I think you want it like this:
LOG.error("writeClosedAccountsToCsvFile: Caught IOException", io);

The logger will know it is logging an exception